gensim/corpora/bleicorpus.py

```diff
@@ -86,14 +111,28 @@ def line2doc(self, line):
 
     @staticmethod
     def save_corpus(fname, corpus, id2word=None, metadata=False):
-        """
-        Save a corpus in the LDA-C format.
-
-        There are actually two files saved: `fname` and `fname.vocab`, where
-        `fname.vocab` is the vocabulary file.
+        """Save a corpus in the LDA-C format.
+
+        Notes
+        -----
+        There are actually two files saved: `fname` and `fname.vocab`, where `fname.vocab` is the vocabulary file.
+
+        Parameters
+        ----------
+        fname : str
+            Path to output filename.
+        corpus : iterable of iterable of (int, float)
+            Input corpus
```

Comment:
Still think that it's not necessary. Also, there's a dot missing at the end of the line.